src/main/java/org/darisadesigns/polyglotlina/ManagersCollections/ConWordCollection.java

```diff
@@ -196,7 +196,7 @@ public ConWord testWordLegality(ConWord word, Map<String, Integer> wordCount, Ma
         if (core.getPropertiesManager().isLocalUniqueness() 
                 && !localWord.isEmpty()
                 && localCount.containsKey(localWord)
-                && localCount.get(localWord) > 1) {
+                && (localCount.containsKey("") || localCount.get(localWord) > 1)) {
```

Comment:
Likewise this one seems strange to me. But it *is* possible to have the natlang equivalent of a conword be blank, which could pose problems.

Reply:
Mhh, I see your point.  If local mandatory is not set there could be several blank locals.  
Maybe then this code needs to be improved since it assumes the current invalid word is already inserted in the conword collection (even if this node has been inserted temporarily).  

That's why the code has to check `>1` instead of just `containsKey`, because if you have 1, then it should be interpreted as duplicate. Currently it needs to check if you have 2 or more to mark it as duplicate.

Reply:
This breaks under these circumstances:
1) Local Uniqueness is set to true
2) A word exists with a blank local value
3) All subsequent words will be marked as non-unique

You're correct in terms of why it works this way, and it's definitely a remnant of some questionable architectural decisions I made when I started PolyGlot, but I'm also wary of changing it without doing some big regression testing over the area. Rewriting this part and the one directly above (along with the code the relies on it) is definitely on my to do list, but is there a problem you're encountering with it right now? If not, I kind of want to kick this can down the road a bit as I work on some other parts.

Let me know what you think there. If it's a roadblock for you, then I'll dig into the best way to refactor both this and the Lexcon UI with some better logic.

Reply:
Haha, I was reverting it as you were writing this :D  
No blocker for the moment, I just updated the UI to update the word before calling testWordLegality and making sure is deleted if it's not legal.  
Might be a good idea to write this as documentation of the function.

Reply:
Sounds good! If this changes, I will probably do something that explicitly breaks the contract here when I do eventually change it. That way you'll get red lines instead of mysterious bugs.

Anyhoo, looks good! Merging this in now!